app/controllers/v0/pension_claims_controller.rb

```diff
@@ -14,64 +17,50 @@ def claim_class
       SavedClaim::Pension
     end
 
-    # rubocop:disable Metrics/MethodLength
     def show
       claim = claim_class.find_by!({ guid: params[:id] }) # will raise ActiveRecord::NotFound
```

Comment:
Consider updating `find_by!({ guid: params[:id] })` to `find_by!(guid: params[:id])`